src/utils/ParseRouteUtils.js

```diff
@@ -56,34 +56,28 @@ function createGpxJson(stops: Array<Object>, startTime: String): Object {
  * path: T[] | string | *, distance: *,
  * routeNumber: (null|*), stops: T[], tripIdentifiers: T[] | string | *}}
  */
-function mergeDirections(first, second) {
-  try {
-    second.stops.shift();
-    second.path.shift();
+function mergeDirections(first, second): Object {
+  second.stops.shift();
+  second.path.shift();
 
-    const path = first.path.concat(second.path);
-    const distance = first.distance + second.distance;
-    const stops = first.stops.concat(second.stops);
-    const tripIDs = first.tripIdentifiers.concat(second.tripIdentifiers);
+  const path = first.path.concat(second.path);
+  const distance = first.distance + second.distance;
+  const stops = first.stops.concat(second.stops);
+  const tripIDs = first.tripIdentifiers.concat(second.tripIdentifiers);
 
-    return {
-      type: first.type,
-      name: first.name,
-      startTime: first.startTime,
-      endTime: second.endTime,
-      startLocation: first.startLocation,
-      endLocation: second.endLocation,
-      path,
-      distance,
-      routeNumber: first.routeNumber,
-      stops,
-      tripIdentifiers: tripIDs,
-    };
-  } catch (error) {
-    throw new Error(
-      LogUtils.logErr(error, { first, second }, 'mergeDirections failed'),
-    );
-  }
+  return {
+    type: first.type,
```

Comment:
alphabetize?

Reply:
classic

Reply:
rip you got me, but i didn't even write this code fml